package.json

```diff
@@ -19,7 +19,9 @@
   },
   "devDependencies": {
     "code": "^1.5.0",
+    "eslint": "^1.8.0",
     "eslint-config-nodesecurity": "^1.1.0",
+    "eslint-plugin-hapi": "^2.0.0",
```

Comment:
Verified that this goes horribly sideways unless **eslint-plugin-hapi** is installed in the top level of node_modules/.
I `rm -rf`ed npm-shrinkwrap.json and node_modules/ and did `npm install && npm run lint` and got the following error after removing **eslint-plugin-hapi** above:

```
$ npm run lint

> nsp@2.0.1 lint /Users/pdehaan/dev/github/nsp2
> eslint .

module.js:338
    throw err;
          ^
Error: Cannot find module 'eslint-plugin-hapi'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at /Users/pdehaan/dev/github/nsp2/node_modules/eslint/lib/cli-engine.js:124:26
    at Array.forEach (native)
    at loadPlugins (/Users/pdehaan/dev/github/nsp2/node_modules/eslint/lib/cli-engine.js:115:21)
    at processText (/Users/pdehaan/dev/github/nsp2/node_modules/eslint/lib/cli-engine.js:202:5)
    at processFile (/Users/pdehaan/dev/github/nsp2/node_modules/eslint/lib/cli-engine.js:262:18)
    at executeOnFile (/Users/pdehaan/dev/github/nsp2/node_modules/eslint/lib/cli-engine.js:671:23)

npm ERR! Darwin 15.0.0
npm ERR! argv "/Users/pdehaan/.nvm/versions/node/v0.12.7/bin/node" "/Users/pdehaan/.nvm/versions/node/v0.12.7/bin/npm" "run" "lint"
npm ERR! node v0.12.7
npm ERR! npm  v2.11.3
npm ERR! code ELIFECYCLE
npm ERR! nsp@2.0.1 lint: `eslint .`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the nsp@2.0.1 lint script 'eslint .'.
npm ERR! This is most likely a problem with the nsp package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     eslint .
npm ERR! You can get their info via:
npm ERR!     npm owner ls nsp
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/pdehaan/dev/github/nsp2/npm-debug.log


$ npm version
{ nsp: '2.0.1',
  npm: '2.11.3',
  http_parser: '2.3',
  modules: '14',
  node: '0.12.7',
  openssl: '1.0.1p',
  uv: '1.6.1',
  v8: '3.28.71.19',
  zlib: '1.2.8' }
```